README.md

```diff
@@ -58,6 +58,8 @@ Browsers don't have native support for Grist [yet :-)] but you can make a little
   * After that, it is just a case of putting a `.grist` file on your server beside this `.html` file, and filing in the options to `bootstrapGrist`.
   * You can also pass `initialData: 'path/to/data.csv'` to import a CSV file into a new table. In this case `initialFile` is optional.
   * You can also pass `elementId: 'element-id` to open Grist within an element in your page.
+    - If using `elementId`, and serving a `.grist` or `.csv` file at a URL that requires cookie-based authentication, be aware that Firefox may not support this yet ([bug report](https://bugzilla.mozilla.org/show_bug.cgi?id=1741489)).
+	- In that case, you can include a small wrapper page for your document as above, and embed it as an iframe yourself.
```

Comment:
So in general will cookies from the parent page be included from within an iframe with the same origin?

Reply:
I think this is the case? In an experiment with `SameSite: Strict` (for which I can confirm seeing the no-cookie problem for a `srcdoc` iframe in Firefox but not Chrome) I see cookies in a regular iframe. This is for pages on the same domain, with `path` set to a common stem.